src/types/events/base-events.ts

```diff
@@ -75,8 +75,7 @@ export type SlackEvent =
   | WorkflowPublishedEvent
   | WorkflowUnpublishedEvent
   | WorkflowStepDeletedEvent
-  | WorkflowStepExecuteEvent
-  | FunctionExecutedEvent;
```

Comment:
Removed due to reverted commit included in history after bad rebase. The changed version should match `main`. To verify: [L78](https://github.com/slackapi/bolt-js/blob/237a503d1e19a8e40ca9ed130dacab1be9752d53/src/types/events/base-events.ts#L78)